modules/core/index.js

```diff
@@ -0,0 +1,10 @@
+export { Connection } from './connection';
+export { Difference } from './difference';
+export { Entity } from './entity';
+export { Graph } from './graph';
+export { History } from './history';
+export { iD_Node as Node } from './node';
```

Comment:
I prefer the first one: 
```js
export { OSMNode as Node } from './node';
```
as long as `Node` `Way` and `Relation` "look" the same, as these are the OSM primitives.

I agree that refactoring all the iD.Node would be a pain and not worth it now.

But why is it ok to export them as `Node` here but not in `./node.js`